### ISSUE TITLE
Contrast for active/hover on the app navigation might be too low

### ISSUE THREAD

**Issue by juliusknorr** [nextcloud/server#33800]
From https://github.com/nextcloud/server/pull/33568#issuecomment-1232657816
contrast quite low, might not pass accessibility test:

<img width="184" alt="image" src="https://user-images.githubusercontent.com/277525/187639327-b30dfa4d-eb96-4fac-94cd-143a62a1133a.png">

@jancborchardt might be a critical one for accessibility, so maybe you can provide a mockup or spec so this can be picked up.

**Comment by jancborchardt** [comment]
How does it look like for focus? As long as we have proper borders there it could be fine.

Also, we could use bold text for the active entry? (Not for hover/focus)

**Comment by ya-d** [comment]
Maybe hover and focus backgrounds should be transparent too, if the entire background is transparent. Looks a bit dull otherwise. Also the perceived color changes quite a lot depended from the color of the background image which is quite irritating. Probably this effect is reduced if hover is transparent too.

Appears almost white:
<img width="311" alt="image" src="https://user-images.githubusercontent.com/30781876/188511070-726b8391-3fd4-42fc-8d13-20c5c96a49b3.png">

Appears gray against bright background:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/30781876/188511112-ca8111a0-8e05-4e6c-959b-1b4db528c6d3.png">

**Comment by szaimen** [comment]
@jancborchardt how can we solve that?

**Comment by susnux** [comment]
I think this is fixed with our a11y fixed for Nextcloud 28/29
